doc/en/user/source/services/wfs/reference.rst

```diff
@@ -304,7 +304,9 @@ For more than one property from a single feature, use a comma-separated list of
 
 While the above permutations for a GetFeature request focused on non-spatial parameters, it is also possible to query for features based on geometry. While there are limited options available in a GET request for spatial queries (more are available in POST requests using filters), filtering by bounding box (BBOX) is supported.
 
-The BBOX parameter allows you to search for features that are contained (or partially contained) inside a box of user-defined coordinates. The format of the BBOX parameter is ``bbox=a1,b1,a2,b2``where ``a1``, ``b1``, ``a2``, and ``b2`` represent the coordinate values. The order of coordinates passed to the BBOX parameter depends on the coordinate system used. (This is why the coordinate syntax isn't represented with ``x`` or ``y``.) To specify the coordinate system, append ``srsName=CRS`` to the WFS request, where ``CRS`` is the Coordinate Reference System you wish to use.
+The BBOX parameter allows you to search for features that are contained (or partially contained) inside a box of user-defined coordinates. The format of the BBOX parameter is ``bbox=a1,b1,a2,b2,[crs]`` where ``a1``, ``b1``, ``a2``, and ``b2`` represent the coordinate values, and ``[crs]`` represents the name of the CRS for the bbox coordinates if it is different to the featureTypes native CRS. The order of coordinates passed to the BBOX parameter depends on the coordinate system used. (This is why the coordinate syntax isn't represented with ``x`` or ``y``.)
```

Comment:
We should be clear that `[crs]` indicates that CRS is optional, since `[` `]` is a common programming convention you did not write down optional.
```
The order of coordinates passed to the BBOX parameter depends on the coordinate system used
(this is why the coordinate syntax isn't represented with ``x`` or ``y``.) The optional ``crs``
parameter is used to name the CRS for the bbox coordinates (if they are different to the
featureTypes native CRS.) 
```